instance/emails.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
```

Comment:
We already get emails when app server builds fail.  How does this new implementation relate to the old one?

Reply:
I didn't know about those e-mails (I'm not in the devops list). Probably some of those e-mails are not needed then. But there's the option of providing more details, or do a better detection of whom to e-mail. E.g. e-mail the person who clicked the blue button. Or find the github username of the PR owner and e-mail their e-mail. Or e-mail also in case of success („build finished“).